### PR TITLE
Patch for PersistentLaunchRecorderTest

### PR DESCRIPTION
**Pull request by mellenburg**
This test would only fail when ordering was changed and would also
segfault the JVM. The segfault was due to an incompatibility with
gradle's native jansi module. This was fixed with explicitly importing
a newer module. The initial failure was due to
PeristentLaunchRecorderTest shadowing the beforeClass method of parent
DefaultCapabilities test suite. Parent classes were made final to avoid
future NPEs in other classes.

### CODE REVIEW

File: sdk/scheduler/src/test/java/com/mesosphere/sdk/state/PersistentLaunchRecorderTest.java [hunk 1]
```diff
@@ -29,7 +29,8 @@
             .build();
 
     @BeforeClass
-    public static void beforeAll() throws Exception {
+    public static void beforeAll2() throws Exception {
+        // Use a new beforeAll name to avoid shadowing parent final void
```
Comment (nickbp) [inline sdk/scheduler/src/test/java/com/mesosphere/sdk/state/PersistentLaunchRecorderTest.java]: - May make sense to just rename the one in OfferEvaluatorTestBase instead. Could see this coming up with other extending classes
- Comment should probably be in javadoc form (`/** foo */`)

File: sdk/scheduler/src/test/java/com/mesosphere/sdk/state/PersistentLaunchRecorderTest.java [hunk 2]
```diff
@@ -28,6 +28,7 @@
             .setSlaveId(TestConstants.AGENT_ID)
             .build();
 
+    /** Use a new beforeAll name to avoid shadowing parent final void */
```
Comment (nickbp) [inline sdk/scheduler/src/test/java/com/mesosphere/sdk/state/PersistentLaunchRecorderTest.java]: should this now be in `DefaultCapabilitiesTestSuite.java`?

Comment (mellenburg) [inline sdk/scheduler/src/test/java/com/mesosphere/sdk/state/PersistentLaunchRecorderTest.java]: D'oh!

Comment (mellenburg) [inline sdk/scheduler/src/test/java/com/mesosphere/sdk/state/PersistentLaunchRecorderTest.java]: The parent function is now a final so it doesnt need to be commented anyway

**Pull request merged**